0.notes.md

```diff
@@ -0,0 +1,1220 @@
+How to identify and reproduce [#1672](https://github.com/elastic/fleet-server/issues/1672)
+----------------------------------------------------------------------------------------------------
+
+TODO(@AndersonQ): delete it before merge
```

Comment:
😲

Reply:
@AndersonQ It would be better to take this content and make it part of https://github.com/elastic/fleet-server/issues/1672

Reply:
I'd put it on the PR description, but it's too long, GitHub does not allow it. So I added it here, but I'll need to delete before merging